src/app/clusters/groupcast-server/GroupcastServer.cpp

```diff
@@ -0,0 +1,105 @@
+#include "GroupcastServer.h"
+
+
+using chip::Protocols::InteractionModel::Status;
+
+namespace chip {
+namespace app {
+namespace Clusters {
+namespace {
+static GroupcastCluster sInstance;
+} // namespace
+
+GroupcastCluster & GroupcastCluster::Instance()
+{
+    return sInstance;
+}
+
+GroupcastCluster::GroupcastCluster() : DefaultServerCluster({ kRootEndpointId, Groupcast::Id })
+{
+}
+
+CHIP_ERROR GroupcastCluster::Startup(ServerClusterContext & context)
+{
+    return CHIP_ERROR_NOT_IMPLEMENTED;
+}
+
+void GroupcastCluster::Shutdown() {}
+
+DataModel::ActionReturnStatus GroupcastCluster::ReadAttribute(const DataModel::ReadAttributeRequest & request, AttributeValueEncoder & encoder)
+{
+    switch (request.path.mAttributeId)
+    {
+    case Groupcast::Attributes::FeatureMap::Id:
+        break;
+    case Groupcast::Attributes::Membership::Id:
+        return mLogic.ReadMembership(request.path.mEndpointId, encoder);
+
+    case Groupcast::Attributes::MaxMembershipCount::Id:
+        return mLogic.ReadMaxMembershipCount(request.path.mEndpointId, encoder);
+    }
+    return Protocols::InteractionModel::Status::UnsupportedAttribute;
+}
+
+DataModel::ActionReturnStatus GroupcastCluster::WriteAttribute(const DataModel::WriteAttributeRequest & request,
+                                        AttributeValueDecoder & decoder)
+{
+    switch (request.path.mAttributeId)
+    {
+    case Groupcast::Attributes::FeatureMap::Id:
+        break;
+    }
+    return Protocols::InteractionModel::Status::UnsupportedAttribute;
+}
+
+CHIP_ERROR GroupcastCluster::Attributes(const ConcreteClusterPath & path, ReadOnlyBufferBuilder<DataModel::AttributeEntry> & builder)
+{
+    return CHIP_ERROR_NOT_IMPLEMENTED;
```

Comment:
Please implement this, even as a skeleton: return the mandatory attributes if nothing else.